client/settings/style.scss

```diff
@@ -0,0 +1,3 @@
+#wc-stripe-account-settings-container {
```

Comment:
The component is not centered on the page.
There's a PR I have open that ensures we have a base layout, which should help in this situation: https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1710

![Screen Shot 2021-08-20 at 2 44 16 PM](https://user-images.githubusercontent.com/273592/130285923-7495bd97-4fa6-43f3-ba5b-a9af60e0bb17.png)

Reply:
Merged that work into this PR. I ignored pretty much all design work on this at first because of #1710 but it's probably better to have it included than have merge conflicts or have to redo stuff after it's merged. 👍